### DISCUSSION TITLE
How to retrieve the Nextcloud logs?

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#5425] [Wiki]
In Nextcloud AIO, the Nextcloud logs are stored inside a docker volume.

If you are the admin of the AIO instance, you can retrieve the logs by so which should allow you to debug internal server errors:
```
sudo docker exec nextcloud-aio-nextcloud cat data/nextcloud.log
```

If you only want to retrieve the latest 10 errors, you can get them by so:
```
sudo docker exec nextcloud-aio-nextcloud tail data/nextcloud.log
```

If you are not an admin, feel free to contact your Nextcloud AIO admin to retrieve the logs for you.

**Comment by AndreDuchesne** [comment]
Erreur interne du serveur
Le serveur est incapable d'exécuter votre requête.

Si cela se reproduit, veuillez envoyer les détails techniques ci-dessous à l'administrateur du serveur.

Le fichier journal du serveur peut fournir plus de renseignements.

[Pour plus de détails, voir la documentation ↗.](https://github.com/nextcloud/all-in-one/discussions/5425)

Renseignements techniques
Adresse distante : 142.115.196.181
ID de la demande : iRN2HAq7t2SgsWXfNiua

**Reply by szaimen** [reply]
Hi, please post this at https://help.nextcloud.com. There more eyes will see this.
